### DISCUSSION TITLE
Как предлагать исправления в документацию?

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#381] [Неопределенное]
Я бы сделал pull request, но не нашел репа с документацией
### Первое

<img src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/aac0f9a3-2cc4-43bc-87dc-4743b4228ed6" width="50%" height="50%">

`--dbname=postgresql://USER:PASSWORD@LOCALCHOST:5432/DB_NAME`

Лишняя "C" в LOCAL**C**HOST, А еще не очень понял с DB_NAME, не помню чтобы базе имя давал, но может память подводит... Убрал просто эту часть и норм работает)

### Второе
<img src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/31f17eff-2a44-4d44-95e4-84c664c13eda" width="50%" height="50%">

Команда 
`bin/totum schema-backup file_dump.sql`
не работает, получаем ошибку 
> In errorException.php line 50: Схема не подключена

Работает только вариант где явно указываем схему через аргумент --schema

Актуально также и для этой части
<img src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/c32f7476-e638-4a7c-888f-23532fb62e98" width="45%" height="45%">

**Comment by ttm-alexeypolunin** [comment]
shema — для `no-multi` установки, сейчас по умолчанию ставятся `multi`. В доках их надо поменять местами, сделаем, когда будет свободное время.

Репо с доками нет — они лежат в большой БД на тотум в которой миллион всего перелинковано и есть автоматизация переводов.

Как ее майнтейнить со стороны — не знаю. Это какая-то 100500 в конце задача.
